tox.ini

```diff
@@ -1,7 +1,11 @@
 [tox]
-envlist = py27,py33
+envlist = django{18,19,110}
```

Comment:
In most of our projects we let tox do the full matrix (python & django version), but both works :) .